record/record.app.js

```diff
@@ -147,6 +148,17 @@
                                 selectable: false
                             };
                             $rootScope.tableModels[i] = model;
```

Comment:
The code changes associated with the comment below should be moved above this line